docs/visio/index.yml

```
@@ -33,15 +33,21 @@ landingContent:
             url: /javascript/api/visio
 
   # Card
-  - title: Resources
+  - title: Visio Developer Resources
```

Same comment as earlier.
```suggestion
  - title: Resources
```